kedro/io/partitioned_data_set.py

```diff
@@ -98,6 +103,7 @@ def __init__(  # pylint: disable=too-many-arguments
         filepath_arg: str = "filepath",
         filename_suffix: str = "",
         credentials: Dict[str, Any] = None,
+        fs_args: Dict[str, Any] = None,
```

Comment:
Sorry for nitpicking, I haven't noticed that the last time I had look, but as `fs_args` is not the last argument in the constructor, then this change technically become a breaking change if someone instantiates the dataset programmatically using positional arguments only. I know it's 99%+ that it's not going to be the case, but just to be on a safe side.

To mitigate this, could you move `fs_args` in the constructor below `load_args` please (for both partitioned and incremental ones)?

Reply:
> Sorry for nitpicking

Not at all! I noticed that the order was different from that of other datasets, and it bothered me, but I didn't want to make unnecessary changes. Sounds like there's a good reason for it! I'll fix it shortly.

Reply:
@DmitriiDeriabinQB Should be good now!